### PR TITLE
Matter 1 3 upmerge

### PR DESCRIPTION
**Pull request by kkasperczyk-no**
Regular Matter fork upmerge to the 1.3.0.0 tag

### CODE REVIEW

**Comment by Damian-Nordic** [comment]
The following change is probably unexpected:
```
+config MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG
+    default n if CHIP_WIFI
+
+config MBEDTLS_SSL_OUT_CONTENT_LEN
+    default 900 if CHIP_WIFI
+
```

**Comment by kkasperczyk-no** [comment]
> The following change is probably unexpected:
> 
> ```
> +config MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG
> +    default n if CHIP_WIFI
> +
> +config MBEDTLS_SSL_OUT_CONTENT_LEN
> +    default 900 if CHIP_WIFI
> +
> ```

Yes, thanks. Btw this PR for now is only for initial testing purposes, so maybe there is no point to waste time on reviewing it. We want to run nightly to early spot the potential problems. It will be rebased soon to the v1.3-branch to pull the cherry-picked commits.

**Comment by Damian-Nordic** [comment]
> Yes, thanks. Btw this PR for now is only for initial testing purposes, so maybe there is no point to waste time on reviewing it. We want to run nightly to early spot the potential problems. It will be rebased soon to the v1.3-branch to pull the cherry-picked commits. Then I will ping you for the final review.
Sure, I don't review such PRs manually, I just did `git diff` with `ncs/master` for Zephyr and nrfconnect directories. :)

File: config/zephyr/Kconfig [hunk 1]
```diff
@@ -515,7 +524,7 @@ config CHIP_PW_RPC
 	bool "Pigweed RPC library"
 	default n
 	help
-	  Links the application with the libraries containing Pigweed RPC
+	  Links the application wCHIP_NXP_PLATFORMith the libraries containing Pigweed RPC
```
Comment (Damian-Nordic) [inline config/zephyr/Kconfig:527]: sth wrong here

Comment (kkasperczyk-no) [reply config/zephyr/Kconfig:527]: Ah good catch. I must have pasted it accidentally while looking for occurrences in the project :)

**Review by ArekBalysNordic — APPROVED** [review]

**Pull request merged**